pkg/diffs/diffs_test.go

```diff
@@ -502,6 +502,138 @@ func TestGetPresubmitsForCiopConfigs(t *testing.T) {
 	}
 }
 
+func podSpecReferencing(info config.Info) *v1.PodSpec {
+	return &v1.PodSpec{
+		Containers: []v1.Container{{
+			Env: []v1.EnvVar{{
+				ValueFrom: &v1.EnvVarSource{
+					ConfigMapKeyRef: &v1.ConfigMapKeySelector{
+						LocalObjectReference: v1.LocalObjectReference{
+							Name: info.ConfigMapName(),
+						},
+						Key: info.Basename(),
+					},
+				},
+			}},
+		}},
+	}
+}
+
+func TestGetImagesPostsubmitsForCiopConfigs(t *testing.T) {
```

Comment:
@petr-muller IMO comparing this to the (nearly identical) tests for the other `Get*submitsForCIopConfigs` function above, this is way clearer